folly/Portability.h

```diff
@@ -284,6 +284,17 @@ inline size_t malloc_usable_size(void* ptr) {
 # define FOLLY_HAS_RTTI 1
 #endif
 
+#if defined(_MSC_VER) && _MSC_FULL_VER <= 190023026 // 2015 RTM or below
```

Comment:
RTM is the release version, it isn't  a prerelease, but there is a good chance that these will be fixed in an update, which is why these are set to go away when the update is released. This is basically future-proofing, as the eventual goal is to remove them completely.